{{ cookiecutter.repo_name }}/.github/workflows/check-news-item.yml

```diff
@@ -0,0 +1,32 @@
+name: Check News Item
+
+on:
+  pull_request_target:
```

Comment:
I am not used to this syntax.  Could you quickly educate me?  How is it different from `pull-request:`.  I am sure it is ok, just a "learning moment" for me.

Reply:
`pull_request`: This event configures the `GITHUB_TOKEN` with read-only permissions by default, especially when triggered by forks. 

`pull_request_target`: This event grants the `GITHUB_TOKEN` write permissions, enabling it to perform actions that modify the repository, such as posting comments (our case here), updating pull request statuses, or merging code.